test/tmetrics.m

```diff
@@ -537,6 +537,40 @@ function testHistogramDelta(testCase)
                 verifyEqual(testCase, str2double(counts{len}), sum(currentvals>bounds(len-1)));
             end
         end
+
+        function testGetSetMeterProvider(testCase)
+            % testGetSetMeterProvider: setting and getting global instance of MeterProvider
+            exporter = opentelemetry.exporters.otlp.OtlpHttpMetricExporter(...
+                "PreferredAggregationTemporality", "Delta");
+            reader = opentelemetry.sdk.metrics.PeriodicExportingMetricReader(exporter, ...
+                "Interval", seconds(2), "Timeout", seconds(1));
+            mp = opentelemetry.sdk.metrics.MeterProvider(reader);
+            setMeterProvider(mp);
+
+            metername = "foo";
+            countername = "bar";
+            m = getMeter(mp, metername);
```

Comment:
This is not calling getMeterProvider. It turns out we are missing a top level getMeter function in the API. You should call the top level getMeter function here which should implicitly call getMeterProvider. See opentelemetry.trace.getTracer.